docs/quick_start/quick_start_with_datagrip.md

```diff
@@ -0,0 +1,40 @@
+# 5.Getting Started With Kyuubi and DataGrip
```

Comment:
the numbers here and below are unnecessary

Reply:
Borrowing from DBeaver，like DBeaver is 2 of quickStart and dataGrip is 5。

Reply:
They are auto generated